.github/workflows/ci-unix-static-av2.yml

```diff
@@ -61,7 +61,7 @@ jobs:
         run: ctest -j $(getconf _NPROCESSORS_ONLN) --output-on-failure
       - name: Check static link bundling
         run: |
-          cc -o avifenc  -I./apps/shared -I./third_party/iccjpeg -I./include apps/avifenc.c \
+          $CC -o avifenc  -I./apps/shared -I./third_party/iccjpeg -I./include apps/avifenc.c \
```

Comment:
Vincent: Thanks for the fix. I am curious how you figured out this is the problem.

Reply:
Simple: on OSX 14, this cannot run because cc does not exist :)